src/daemon/main.cpp

```diff
@@ -207,7 +207,18 @@ int main(int argc, char const * argv[])
     bf::path log_file_path {data_dir / std::string(CRYPTONOTE_NAME ".log")};
     if (! vm["log-file"].defaulted())
       log_file_path = command_line::get_arg(vm, daemon_args::arg_log_file);
-    log_file_path = bf::absolute(log_file_path, relative_path_base);
+
+    if (log_file_path == "syslog")
```

Comment:
this block shouldn't be compiled with disabled syslog - there's no reasons user can't have log file named "syslog"